docs/conf/user-interface/index.md

```diff
@@ -0,0 +1,621 @@
+| `--bui-border`          | It should be used on top of `--bui-bg-surface-1`.   |
+| `--bui-border-hover`    | Used when the component is interactive and hovered. |
+| `--bui-border-pressed`  | Used when the component is interactive and hovered. |
+| `--bui-border-disabled` | Used when the component is disabled.                |
+| `--bui-border-danger`   | It should be used on top of `--bui-bg-danger`.      |
+| `--bui-border-warning`  | It should be used on top of `--bui-bg-warning`.     |
+| `--bui-border-success`  | It should be used on top of `--bui-bg-success`.     |
+
+#### Special colors
+
+These colors are used for special purposes like ring, scrollbar, ...
+
+| Token Name              | Description                       |
+| ----------------------- | --------------------------------- |
+| `--bui-ring`            | The color of the ring.            |
+| `--bui-scrollbar`       | The color of the scrollbar.       |
+| `--bui-scrollbar-thumb` | The color of the scrollbar thumb. |
+
+#### Font families
+
+We have two fonts that we use across Backstage UI. The first one is the sans-serif font that we use for the body of the application. The second one is the monospace font that we use for code blocks and tables.
+
+| Token Name           | Description                        |
+| -------------------- | ---------------------------------- |
+| `--bui-font-regular` | The sans-serif font for the theme. |
+| `--bui-font-mono`    | The monospace font for the theme.  |
+
+#### Font weights
+
+We have two font weights that we use across Backstage UI. Regular or Bold.
+
+| Token Name                  | Description                            |
+| --------------------------- | -------------------------------------- |
+| `--bui-font-weight-regular` | The regular font weight for the theme. |
+| `--bui-font-weight-bold`    | The bold font weight for the theme.    |
+
+#### Spacing
+
+We built a spacing system based on a single value `--bui-space`. This value is used to calculate the spacing for all the components. By default if you would like to increase or decrease the spacing between your components you can do it simply by updating `--bui-space` and it will apply to all spacing values.
+
+`--bui-space` is not used directly in any components but serve as an easy way to calculate the other values.
+
+| Token Name    | Description                                                       |
+| ------------- | ----------------------------------------------------------------- |
+| `--bui-space` | The base unit for the spacing system. Default value is `0.25rem.` |
+
+#### Radius
+
+We use a radius system to make sure that the components have a consistent look and feel.
+
+| Token Name          | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| `--bui-radius-1`    | The radius of the component. Default value is `0.125rem`. |
```

Comment:
How come radius doesn't have a base value that you can set, like spacing?

Reply:
It did not felt necessary with such a small amount of tokens. But happy to explore a different path.